config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -485,9 +490,464 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
+                          ],
+                          [ a                 form:RequiredConstraint ;
+                            form:grouping     form:Bag ;
+                            sh:resultMessage  "Dit veld is verplicht." ;
+                            sh:path           lblodSubsidie:projectOnderdeelNaam            
+                          ] ;
+        form:displayType  displayTypes:defaultInput ;
+        sh:group          ext:projectOnderdeelPg-sub .
+
+
+    ##########################################################
+    # Field: Projectonderdeel toelichting Input
+    ##########################################################
+
+    ext:projectOnderdeelBeschrijvingF
+        a                 form:Field ;
+        sh:name           "Geef een korte toelichting over wat dit projectonderdeel inhoudt" ;
+        sh:order          1 ;
+        sh:path           lblodSubsidie:projectOnderdeelBeschrijving ;
+        form:validations
+                          [ a                 form:MaxLength ;
+                            form:grouping     form:MatchEvery ;
+                            form:max          "750" ;
+                            sh:resultMessage  "Max. karakters overschreden." ;
+                            sh:path            lblodSubsidie:projectOnderdeelBeschrijving
+                          ],
+                          [ a                 form:RequiredConstraint ;
+                            form:grouping     form:Bag ;
+                            sh:resultMessage  "Dit veld is verplicht." ;
+                            sh:path           lblodSubsidie:projectOnderdeelBeschrijving            
+                          ] ;
+        form:displayType  displayTypes:textArea ;
+        sh:group          ext:projectOnderdeelPg-sub .
+
+
+    ##########################################################
+    # Listing: Projectonderdeel Timing
+    ##########################################################
+        ##########################################################
+        #  property-group
+        ##########################################################
+        ext:projectOnderdeelTimingListingPg a form:PropertyGroup;
+            sh:name "Timing" ;
+            form:help """Geef de verschillende stappen aan, zoveel mogelijk in chronologische volgorde, en duidt in de laatste twee kolommen de start- en einddatum aan.""";
+            sh:order 1 .
+
+        ##########################################################
+        #  listing-scope: timing
+        ##########################################################
+        ext:projectOnderdeelTimingListingS a form:Scope;
+          sh:path lblodSubsidie:projectOnderdeelTimingUnit.
+
+        ##########################################################
+        # Listing: Projectonderdeel timing listing
+        ##########################################################
+        ext:projectOnderdeelTimingL a form:Listing;
+          a form:ListingTable;
```

Comment:
I would add a `form:noDataMessage "";` to all listing tables, so it doesn't show "No items" when the table is empty.

Reply:
@aatauil I think one still "needs" to be done (or the commits weren't pushed yet).